typescript/sdk/src/chains.ts

```diff
@@ -4,6 +4,35 @@ import { ethers } from 'ethers';
 import { IChainConnection } from './provider';
 import { ChainMap, ChainName } from './types';
 
+export const ethereum: IChainConnection = {
+  provider: new ethers.providers.JsonRpcProvider(
+    'https://mainnet-nethermind.blockscout.com',
+    1,
+  ),
+  confirmations: 7,
+};
+
+export const celo: IChainConnection = {
+  provider: new StaticCeloJsonRpcProvider('https://forno.celo.org', 42220),
+  confirmations: 1,
+};
+
+export const polygon: IChainConnection = {
+  provider: new ethers.providers.JsonRpcProvider(
+    'https://rpc-mainnet.matic.quiknode.pro',
+    137,
+  ),
+  confirmations: 200,
```

Comment:
fwiw I've seen deeper reorgs than this, but this is for just non-core use right? in other words, does this # ever get passed in as config to the agent deployments done by `deploy`?

Reply:
It does look like we now source the number of confirmations from `AgentConfig`, though I would have intuitively thought we source them from the same place. IMO we should, wdyt @mattiecnvr @webbhorn ? 

I thought 200 was the max reorg depth for polygon, what do you think is the right safe period?

Reply:
https://twitter.com/koeppelmann/status/1452603141971759106?lang=en

Reply:
Yes, we should be consistent on where the information is defined. Easier to keep track of.